vcr/errors.py

```diff
@@ -3,8 +3,7 @@ class CannotOverwriteExistingCassetteException(Exception):
 
 
 class UnhandledHTTPRequestError(KeyError):
-    '''
-    Raised when a cassette does not c
-    ontain the request we want
-    '''
+    """
+    Raised when a cassette does not contain the request we want
```

Comment:
We've been trying to make this more python standard compliant:
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings